### PR TITLE
Fix corporatedomains whois expires_on date format

### PR DESCRIPTION
**Pull request by njng**
Fix **expires_on** for corporatedomains WHOIS parser when the date format is **MM/DD/YYYY HH:MM:SS** by changing the format before it is passed into `parse_time`. Note that there are still 2 different formats that could be possible for **expires_on**, so we only change the slash-formatted dates.

### CODE REVIEW

File: lib/whois/parsers/whois.corporatedomains.com.rb [hunk 1]
```diff
@@ -22,6 +22,39 @@ class WhoisCorporatedomainsCom < BaseIcannCompliant
...
+
+      # There are 2 different date formats which are used for creation and expiration dates:
+      #   - hyphen-separated date format YYYY-MM-DDTHH:MM:SSZ
+      #   - slash-separated date format MM/DD/YYYY HH:MM:SS
+      #
+      # The latter format causes problems because `parse_time` parsed as (DD/MM/YYYY HH:MM:SS)
+      def reformat_slash_date_format(value)
+        date, time = value.split
+        month, day, year = date.split("/")
+        reformatted_date = [year, month, day].join("-")
```
Comment (itghlu) [inline lib/whois/parsers/whois.corporatedomains.com.rb]: `Time.utc(year, month, day)` will give you a Time object which you can then convert to a string with `to_s` or `as_json`. The manual array join and string interpolation is not needed then.

Comment (njng) [reply lib/whois/parsers/whois.corporatedomains.com.rb]: If I'm already creating a Time object, it doesn't seem like there's any reason to convert it to a string, add the time, and then pass it in to `parse_time`, is there?

Comment (itghlu) [reply lib/whois/parsers/whois.corporatedomains.com.rb]: Yeah just returning the Time you generate seems good.

**Comment by itghlu** [comment]
LGTM

**Pull request merged**